### PR TITLE
Add styling to WinModal

### PR DESCRIPTION
**Pull request by Enessetere**

### CODE REVIEW

File: src/Animals/Cow.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ export class Cow extends Animal {
   constructor() {
     super(
       AnimalNames.COW,
-      './static/images/avatars/cow.png',
+      './static/images/avatars/cow.svg',
```
Comment (vieraboschkova) [inline src/Animals/Cow.ts:9]: All these paths are coming from the configuration, not the animals itself.

File: src/app/components/WinModal.ts [hunk 2]
```diff
@@ -1,38 +1,53 @@
...
   }
 
   createWinModal(player: Player): HTMLElement {
-    this.renderBasicModal(
-      'CONGRATULATIONS!',
-      `${player.theName} has won!`,
-      this.createImage(player),
+    const heading = Render.elementFactory(
+      'h2',
+      { className: 'modal__win-heading' },
```
Comment (vieraboschkova) [inline src/app/components/WinModal.ts]: Perhaps we could have a modal__heading class which would be the same for all the modals?

File: src/app/components/WinModal.ts [hunk 3]
```diff
@@ -1,38 +1,53 @@
...
-      alt: `${theName}-avatar`,
-    });
+    const text = Render.elementFactory(
+      'div',
+      { className: 'modal__win-text' },
+      `${player.theName} wins`,
+    );
+    const button = Render.elementFactory(
+      'button',
+      { type: 'submit', className: 'modal__win-button' },
```
Comment (vieraboschkova) [inline src/app/components/WinModal.ts]: Is it necessary for this button to be a  submit button?

File: styles/components/_modal.scss [hunk 4]
```diff
@@ -12,12 +12,12 @@
...
   // Color /
-  background-color: rgba(0, 0, 0, 0.8);
+  background-color: rgba(0, 0, 0, 0.5);
   &__container {
     // Display & Box Model /
     display: flex;
-    width: 80%;
-    height: 80%;
+    // width: 80%;
+    // height: 80%;
```
Comment (vieraboschkova) [inline styles/components/_modal.scss]: So the modal will always be depending on the content, not on the size of the screen?

File: styles/components/_modal.scss [hunk 5]
```diff
@@ -39,4 +39,45 @@
   }
   &__button {
   }
+
+  &__win-heading {
```
Comment (vieraboschkova) [inline styles/components/_modal.scss]: consider modal__heading--win (if modal__heading would be basic for all modals, but this one could have its perks in here)

File: styles/components/_modal.scss [hunk 6]
```diff
@@ -39,4 +39,45 @@
...
+    font-size: 2.5rem;
+  }
+
+  &__win-image {
```
Comment (vieraboschkova) [inline styles/components/_modal.scss]: maybe case modal__image--win? We could join all the modal styles in one file later.

File: styles/components/_modal.scss [hunk 7]
```diff
@@ -39,4 +39,45 @@
...
+  }
+
+  &__win-medal {
+    position: absolute;
+    right: -2rem;
+    bottom: -2rem;
+    height: 6rem;
+  }
+
+  &__win-text {
```
Comment (vieraboschkova) [inline styles/components/_modal.scss]: modal__text(--win?), the same basic text styling in all modals in the game

File: styles/components/_modal.scss [hunk 8]
```diff
@@ -39,4 +39,45 @@
   }
   &__button {
   }
+
+  &__win-heading {
+    font-size: 2.5rem;
```
Comment (vieraboschkova) [inline styles/components/_modal.scss]: We need a font-family also.

File: styles/components/_modal.scss [hunk 9]
```diff
@@ -39,4 +39,45 @@
...
+    height: 6rem;
+  }
+
+  &__win-text {
+    margin-top: 2rem;
+    color: #71cfd7;
+    font-size: 2rem;
+  }
+
+  &__win-button {
```
Comment (vieraboschkova) [inline styles/components/_modal.scss]: it is the same button as on the landing page. Perhaps you could talk it over with @paula0403 and figure out a styling together? It also needs to have the border removed (grey and black) and a separate styling for hover and focus (remove the black outline).

**Review by vieraboschkova — CHANGES_REQUESTED** [review]
Consider different naming convention, get in touch with the person who is preparing Landing page (common styling), add font family and styles for hovering and focus. Perhaps see if you can reuse classes from other modals or unify them.

File: src/Animals/Cow.ts (hunk 1 above)
Comment (Enessetere) [reply src/Animals/Cow.ts:9]: Yes, but until we will use configuration in every element, it will be better to keep this for now.

File: styles/components/_modal.scss (hunk 4 above)
Comment (Enessetere) [reply styles/components/_modal.scss]: I think this will look better. I.e. on win modal there would be large blank space around if it would depend on screen size. And if we make it smaller, then trade modal can be squeezed.

File: styles/components/_modal.scss (hunk 8 above)
Comment (Enessetere) [reply styles/components/_modal.scss]: I will add Roboto 700 and 400 in next batch.

File: src/app/components/WinModal.ts (hunk 2 above)
Comment (Enessetere) [reply src/app/components/WinModal.ts]: done

File: src/app/components/WinModal.ts (hunk 3 above)
Comment (Enessetere) [reply src/app/components/WinModal.ts]: changed to button

File: styles/components/_modal.scss (hunk 5 above)
Comment (Enessetere) [reply styles/components/_modal.scss]: removed this class and content moved to modal__heading

File: styles/components/_modal.scss (hunk 7 above)
Comment (Enessetere) [reply styles/components/_modal.scss]: done

File: styles/components/_modal.scss (hunk 6 above)
Comment (Enessetere) [reply styles/components/_modal.scss]: done

File: styles/components/_modal.scss (hunk 9 above)
Comment (Enessetere) [reply styles/components/_modal.scss]: For now all settings will be added in this specific class, after Landing Page will be merged I will change it. (added comment about this)

File: src/Animals/Cow.ts (hunk 1 above)
Comment (vieraboschkova) [reply src/Animals/Cow.ts:9]: 👍

File: styles/components/_modal.scss (hunk 4 above)
Comment (vieraboschkova) [reply styles/components/_modal.scss]: Well, I guess it would be important if we were considering becoming more "RWD", cause then breakpoints would be important. I thought it would be better for all modals to be the same, but you have a point, the win modal has very little content in comparison to the others.

File: styles/components/_modal.scss (hunk 9 above)
Comment (vieraboschkova) [reply styles/components/_modal.scss]: 👍

**Review by synowa — APPROVED** [review]

File: src/app/manuals/herdManual.ts [hunk 10]
```diff
@@ -8,50 +8,50 @@ export function herdDemo(): void {
     {
       name: AnimalNames.RABBIT,
       tradeValue: 1,
-      path: '/static/images/avatars/rabbit.png',
+      path: './static/images/avatars/rabbit.svg',
```
Comment (synowa) [inline src/app/manuals/herdManual.ts:11]: IMO, it will be good for such changes to be done in a separate PR with e.g. 'fix' label.

**Review by synowa — APPROVED** [review]

**Review by vieraboschkova — APPROVED** [review]
